wp-content/themes/core/blocks/tribe/terms/style.pcss

```diff
@@ -0,0 +1,82 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+/* -----------------------------------------------------------------------
+ * Terms Block
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-terms {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+
+	&.aligncenter {
+		justify-content: center;
+	}
+
+	&.alignright {
+		justify-content: flex-end;
+	}
+}
+
+/* -----------------------------------------------------------------------
+ * Terms Block - List
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-terms__list {
+	display: inline-flex;
+	align-items: center;
+	gap: 8px;
```

Comment:
SHould we be using a spacer var for this?

Reply:
Updated this to a spacer variable.